third_party/move/move-compiler-v2/tests/v1.unmatched

```diff
@@ -265,6 +261,7 @@ move-compiler/tests/move_check/typing/{
   declare_pop_resource.move,
   entry_signature_no_warning.move,
   extraneous_acquire.move,
+  global_invalid.move,
```

Comment:
It is here because global_invalid in V2 is placed in the `bytecode_generator` folder